_sass/components/_layout.scss

```diff
@@ -64,16 +64,35 @@ section > .container {
 }
 
 @media #{$desktop} {
+  div > .container,
+  section > .container {
+    padding-left: 1.5rem;
+    padding-right: 1.5rem;
+  }
```

Comment:
What's the intention with these styles? Given how common `.container` would be, it feels like it could be fragile, especially if adding wrapping `div`'s to containers in future revisions. As seen with styles below, it also requires quite a few compensations (with positioned elements especially, like `usa-search`).

Is it expected to add a padding for the "One account" partial? In that case, I think a `margin-x-3` on the `tagline-content` would be preferable.

Is it expected to reduce the default container width? In that case, I think [adjusting `$container-width`](https://github.com/18F/identity-site/blob/6294a653bd5abc2bf6ecaad593594744691a162b/_sass/_variables.scss#L107) would be preferable.

Reply:
Talked to @aduth and we will re-visit this as we look at upgrading the USWDS to the latest version that is leveraged in the login.gov design system.